src/core-layers/solid-polygon-layer/polygon-tesselator.js

```diff
@@ -114,11 +106,11 @@ function getPolygonOffsets(polygons) {
 // Returns the offset of each hole polygon in the flattened array for that polygon
 function getHoleIndices(complexPolygon) {
   let holeIndices = null;
-  if (count(complexPolygon) > 1) {
+  if (complexPolygon.length > 1) {
```

Comment:
Just a thought - If we drop `count`, there is no way for the user to support immutable data himself through accessors.

Reply:
These polygons are returned by the `getPolygon` accessor. The user will need to make sure their accessor returns a plain array.